src/components/auth/Signin.vue

```diff
@@ -1,6 +1,11 @@
 <template lang="pug">
     v-card(class="pa-10 ma-auto text-center" width="300")
         h2(class="mb-8") Welcome.
+        <div v-if="errorMessage">
+            <v-alert color="red">
```

Comment:
Change this to use the error theming as stated here. https://vuetifyjs.com/en/features/theme/#light-and-dark. This makes it easier to switch colors and not have to change it in a bunch of places